sources/packages/forms/src/form-definitions/studentdisabilityappeal.json

```diff
@@ -0,0 +1,369 @@
+      "refreshOnChange": false,
+      "customClass": "",
+      "hidden": false,
+      "modalEdit": false,
+      "key": "disabilityInformationTitle",
+      "tags": [],
+      "properties": {},
+      "conditional": {
+        "show": null,
+        "when": null,
+        "eq": "",
+        "json": ""
+      },
+      "customConditional": "",
+      "logic": [],
+      "attributes": {},
+      "overlay": {
+        "style": "",
+        "page": "",
+        "left": "",
+        "top": "",
+        "width": "",
+        "height": ""
+      },
+      "type": "htmlelement",
+      "input": false,
+      "tableView": false,
+      "placeholder": "",
+      "prefix": "",
+      "suffix": "",
+      "multiple": false,
+      "defaultValue": null,
+      "protected": false,
+      "unique": false,
+      "persistent": false,
+      "clearOnHide": true,
+      "refreshOn": "",
+      "redrawOn": "",
+      "dataGridLabel": false,
+      "labelPosition": "top",
+      "description": "",
+      "errorLabel": "",
+      "tooltip": "",
+      "hideLabel": false,
+      "tabindex": "",
+      "disabled": false,
+      "autofocus": false,
+      "dbIndex": false,
+      "customDefaultValue": "",
+      "calculateValue": "",
+      "calculateServer": false,
+      "widget": null,
+      "validateOn": "change",
+      "validate": {
+        "required": false,
+        "custom": "",
+        "customPrivate": false,
+        "strictDateValidation": false,
+        "multiple": false,
+        "unique": false
+      },
+      "allowCalculateOverride": false,
+      "encrypted": false,
+      "showCharCount": false,
+      "showWordCount": false,
+      "allowMultipleMasks": false,
+      "addons": [],
+      "id": "eg5zawe"
+    },
+    {
+      "title": "Disability Information panel",
+      "labelWidth": "",
+      "labelMargin": "",
+      "theme": "default",
+      "tooltip": "",
+      "customClass": "",
+      "collapsible": false,
+      "hidden": false,
+      "hideLabel": true,
+      "disabled": false,
+      "modalEdit": false,
+      "key": "disabilityInformationPanel",
+      "tags": [],
+      "properties": {},
+      "customConditional": "",
+      "conditional": {
+        "json": "",
+        "show": null,
+        "when": null,
+        "eq": ""
+      },
+      "logic": [],
+      "attributes": {},
+      "overlay": {
+        "style": "",
+        "page": "",
+        "left": "",
+        "top": "",
+        "width": "",
+        "height": ""
+      },
+      "type": "panel",
+      "label": "Panel",
+      "breadcrumb": "default",
+      "tabindex": "",
+      "input": false,
+      "tableView": false,
+      "components": [
+        {
+          "autofocus": false,
+          "input": true,
+          "tableView": true,
+          "inputType": "radio",
+          "label": "Are you a student with disabilities and wish to be assessed for disability funding?",
+          "key": "disabilityInformation",
```

Comment:
I would prefer the api key to follow the existing appeal format like this. e.g `studentNewPDPPDStatus`

![image](https://github.com/bcgov/SIMS/assets/54600590/37a5a08d-a25b-4317-ab28-0549700a6909)